database/factories/ModelFactory.php

```diff
@@ -140,6 +140,7 @@
         'reportback' => 1,
         'civic_action' => 1,
         'scholarship_entry' => 1,
+        'anonymous' => 0,
```

Comment:
Once we add casts for these, let's just set these as `true` or `false` here rather than integers.